src/Page/NotAlone.elm

```diff
@@ -29,6 +32,79 @@ update msg model =
 view : Model -> Html Msg
 view model =
     div []
-        [ div [] [ text "Not Alone page" ]
+        [ header []
+            [ h1 [ css [ pageHeading ] ] [ text "[cCc] You Are Not Alone" ]
+            ]
+        , div [ css [ grid ] ]
+            [ card "[cCc] A medium quote about a recognisable experience. A medium quote about a recognisable experience." "My Name" "22"
+            , card "[cCc] A short quote about a recognisable experience" "My Name" "33"
+            , card "[cCc] A medium quote about a recognisable experience. A medium quote about a recognisable experience." "My Name" "62"
+            , card "[cCc] A long quote about a recognisable experience. A long quote about a recognisable experience, A long quote about a recognisable experience." "My Name" "33"
+            , card "[cCc] A short quote about a recognisable experience" "My Name" "55"
+            , card "[cCc] A long quote about a recognisable experience. A long quote about a recognisable experience, A long quote about a recognisable experience." "My Name" "44"
+            ]
+        , div [] [ text "Not Alone page" ]
         , a [ href "definition" ] [ text "Find out more about Economic Abuse" ]
         ]
+
+
+grid : Style
+grid =
+    batch
+        [ displayFlex
+        , flexWrap wrap
+        , justifyContent spaceBetween
+        ]
+
+
+card : String -> String -> String -> Html msg
+card quote name age =
+    div
+        [ css
+            [ cardStyle
+            , withMedia [ only screen [ Media.minWidth (px 576) ] ]
+                [ flex3 zero zero desktopWidth ]
+            ]
+        ]
+        [ span [ css [ quoteStyle ] ] [ text quote ]
+        , span [ css [ details ] ] [ text ("- " ++ name ++ ", " ++ age) ]
+        ]
+
+
+desktopWidth =
+    calc (pct 50) minus (rem 1)
+
+
+mobileWidth =
+    calc (pct 100) minus (rem 1)
+
+
+cardStyle : Style
+cardStyle =
+    batch
+        [ border3 (px 2) solid Theme.colours.grey
+        , borderRadius (rem 1)
+        , flex3 zero zero mobileWidth
+        , displayFlex
+        , flexDirection column
+        , justifyContent center
+        , margin2 (rem 1) zero
+        , padding (rem 1)
+        , Css.minHeight (px 200)
+        , textAlign center
+        ]
+
+
+quoteStyle : Style
+quoteStyle =
+    batch
+        [ Css.flex3 zero zero (pct 60)
+        ]
+
+
+details : Style
```

Comment:
`detailsStyle` for consistancy?

Reply:
This is a bit annoying. I had tried to not pad all the style functions out with -Style suffixes before but a lot of names are already in use.

Reply:
I can't see a particular Elm convention for function naming based on return type - should we decide one? Would it be easier if all style functions ended `Style`?

Reply:
Yeah - I was just going with the convention you set by the 2 above it.

Reply:
There will be styles in Theme and Definition that don't, but I can make everything have it.